src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/LicenseAcceptanceWindow.xaml

```diff
@@ -99,82 +98,73 @@
       </DataTemplate>
     </ResourceDictionary>
   </Window.Resources>
-  <Grid>
-    <Grid.RowDefinitions>
-      <RowDefinition
+  <ScrollViewer x:Name="sv" CanContentScroll="False" VerticalScrollBarVisibility="Auto" HorizontalScrollBarVisibility="Auto">
+    <Grid x:Name="_licenseAcceptanceGrid" MinWidth="{Binding ElementName=sv, Path=ViewportWidth}" MinHeight="{Binding ElementName=sv, Path=ViewportHeight}"
+          Width="{Binding ElementName=sv, Path=ViewportWidth}" Height="{Binding ElementName=sv, Path=ViewportHeight}">
+      <Grid.RowDefinitions>
+        <RowDefinition
         Height="Auto" />
-      <RowDefinition
+        <RowDefinition />
+        <RowDefinition
         Height="Auto" />
-      <RowDefinition />
-      <RowDefinition
+        <RowDefinition
         Height="Auto" />
-      <RowDefinition
-        Height="Auto" />
-    </Grid.RowDefinitions>
-    <TextBlock
-      Grid.Row="0"
-      Grid.Column="0"
-      Margin="12,8,12,0"
-      x:Name="_licenseAcceptanceText"
-      AutomationProperties.Name="{Binding Text}"
-      Text="{x:Static nuget:Resources.Text_LicenseAcceptance}"
-      FontSize="{Binding ElementName=_self,Path=FontSize,Converter={StaticResource Font122PercentSizeConverter}}" />
-
-    <TextBlock
-      Grid.Row="1"
-      Grid.Column="0"
-      Margin="12,4,12,0"
-      x:Name="_licenseHeaderText"
-      AutomationProperties.Name="{Binding Text}"
-      Text="{x:Static nuget:Resources.Text_LicenseHeaderText}"
-      TextWrapping="Wrap" />
-    <ItemsControl
-      Grid.Row="2"
-      Grid.Column="0"
-      Margin="12,8"
-      MinHeight="130"
-      IsTabStop="False"
-      ItemsSource="{Binding}"
-      ItemTemplate="{StaticResource LicenseItemTemplate}">
-      <ItemsControl.Template>
-        <ControlTemplate
-          TargetType="{x:Type ItemsControl}">
-          <Border
-            BorderThickness="1"
-            BorderBrush="{DynamicResource {x:Static nuget:Brushes.BorderBrush}}">
-            <ScrollViewer
-              Padding="3"
-              CanContentScroll="True"
-              VerticalScrollBarVisibility="Visible"
-              HorizontalScrollBarVisibility="Disabled"
-              AutomationProperties.LabeledBy="{Binding ElementName=_licenseHeaderText}">
-              <ItemsPresenter></ItemsPresenter>
-            </ScrollViewer>
-          </Border>
-        </ControlTemplate>
-      </ItemsControl.Template>
-    </ItemsControl>
-
-    <TextBlock
+      </Grid.RowDefinitions>
+      <Grid Grid.Row="0" Width="{Binding Width, ElementName=_licenseAcceptanceGrid}">
+        <TextBlock
+        Margin="12,4,12,0"
+        x:Name="_licenseHeaderText"
+        AutomationProperties.Name="{Binding Text}"
+        Text="{x:Static nuget:Resources.Text_LicenseHeaderText}"
+        TextWrapping="Wrap" />
+      </Grid>
+      <Grid Grid.Row="1" Width="{Binding Width, ElementName=_licenseAcceptanceGrid}">
+        <ItemsControl
+        Margin="12,8"
+        MinHeight="116"
```

Comment:
Brought down from 130 so it'll render at minimum supported resolution and maximum supported scaling.